dll/win32/rpcrt4/rpc_async.c

```diff
@@ -158,3 +158,50 @@ RPC_STATUS WINAPI RpcAsyncCancelCall(PRPC_ASYNC_STATE pAsync, BOOL fAbortCall)
     FIXME("(%p, %s): stub\n", pAsync, fAbortCall ? "TRUE" : "FALSE");
     return RPC_S_INVALID_ASYNC_HANDLE;
 }
+
+#ifdef __REACTOS__
+/***********************************************************************
+ *           RpcGetAuthorizationContextForClient [RPCRT4.@]
+ * 
+ * Called by RpcFreeAuthorizationContext to return the Authz context.
+ * 
+ * PARAMS
+ *  ClientBinding        [I] Binding handle, represents a binding to a client on the server.
+ *  ImpersonateOnReturn  [I] Directs this function to be represented the client on return.
+ *  Reserved1            [I] Reserved, equal to null.
+ *  expiration_time      [I] Points to the exact date and time when the token expires.
+ *  Reserved2            [I] Reserved, equal to a LUID structure which has a members,
+ *                           each of them is set to zero.
+ *  Reserved3            [I] Reserved, equal to zero.
+ *  Reserved4            [I] Reserved, equal to null.
+ *  authz_client_context [I] Points to an AUTHZ_CLIENT_CONTEXT_HANDLE structure
+ *                           that has direct pass to Authz functions.
+ * 
+ * RETURNS
+ *  Success: RPC_S_OK.
+ *  Failure: Any error code.
+ */
+RPC_STATUS
+WINAPI
+RpcGetAuthorizationContextForClient(RPC_BINDING_HANDLE ClientBinding,
+                                    BOOL ImpersonateOnReturn,
+                                    void * Reserved1,
+                                    PLARGE_INTEGER expiration_time,
+                                    LUID Reserved2,
+                                    DWORD Reserved3,
+                                    PVOID Reserved4,
+                                    PVOID *authz_client_context)
+{
+    FIXME("(%p, %s, %p, %p, (%u, %d), %u, %p, %p): stub\n",
```

Comment:
Serge's suggestion for this line was under the assumption that all his other comments would also be applied, so it doesn't stand on its own.
The `%s` makes no sense for a `BOOL` argument, it should be `%d`.
`(%u, %d)` should now be `(%d, %u)`.

Reply:
Fixed.